cmd/admin-handlers-config-kv.go

```diff
@@ -402,7 +410,16 @@ func (a adminAPIHandlers) SetConfigHandler(w http.ResponseWriter, r *http.Reques
 
 	// Make sure to write backend is encrypted
 	if globalConfigEncrypted {
```

Comment:
@harshavardhana @klauspost This seems questionable to me.
When the server starts and the backend encryption is not completed then the server will continue encrypting/migrating
all config data until the backend encryption is completed. If this where not the case then the server would start with an
inconsistent backend configuration. 

Then then the backend encryption has to be in the state `complete` hee, right?!. So, there is no point in writing the state `complete` again?!

Reply:
@aead this code is not needed but it was added for situations where a user has dropped into safe mode and we want to recover the server. 

The problem is it is hard to provide atomicity with disconnected operations.